site/plugins.json

```
@@ -724,6 +724,6 @@
     "name": "Netlify Emails",
     "package": "@netlify/plugin-emails",
     "repo": "https://github.com/netlify/plugins",
```

```suggestion
    "repo": "https://github.com/netlify/netlify-emails-plugin",
```
just noticed this repo link wasn't pointing to the plugin repo (not sure you want to add it to this PR but just in case)

---

So it turns out, the`https://github.com/netlify/plugins` is a special url. We found this out the hard way, but any internal repo should point to `https://github.com/netlify/plugin`s. Not actually entirely sure why.

---

To be more useful, here's a slack thread https://netlify.slack.com/archives/C037WKGL023/p1665154643214539